api/src/main/resources/workbench.yaml

```diff
@@ -694,6 +694,26 @@ paths:
           schema:
             $ref: "#/definitions/EmptyResponse"
 
+  # Note: all requests tagged as "cron" must have the header X-Appengine-Cron:
+  # true, which app engine itself only sets when invoking as a cronjob.
+  # See https://cloud.google.com/appengine/docs/standard/java/config/cron#securing_urls_for_cron
+
+  /v1/cron/auditBigQuery:
+    get:
+      security: []
+      tags:
+        - audit
+        - cron
```

Comment:
So is the deal that the first tag is used to determine what service the method is in, and subsequent tags can be whatever we want?

Wondering if we could somehow use security for this instead... or there's always adding a @Cron annotation to methods in the controller.

Anyway, I like that this is in Swagger, just noodling over the options.

Reply:
Yes. Security as we're currently using it for oauth is a slight abuse since it can't be used as such without more information (see TODO I put in securities above a while back). Similarly I'm not sure that formalizing this header requirement as a security here (in a way that clients couldn't actually use) is better - closest thing would be an `apiKey` which might be misleading.

Since I need to disable normal auth here, I'd prefer to keep the annotation in Swagger somewhere (either as tag or security).